admin/src/components/UserTable.vue

```diff
@@ -72,6 +72,14 @@
         </b-button>
       </template>
 
+      <template #cell(has_elopage)="row">
+        <b-button :variant="row.item.hasElopage ? 'success' : 'danger'" size="md" class="mr-2">
```

Comment:
Graphql does not query for hasElopage. So this is always undefined and therefore false.

Reply:
> Graphql does not query for hasElopage. So this is always undefined and therefore false.

yes, i have agreed with @Elweyn. that i will first design a forntend query so that this **hasElopage** is available ... this will be implemented with hannes his work in the backend with the next step from the backend.

Reply:
Please do not merge unfinished not working partial work - this makes things broken. Make sure to only merge working packages

Reply:
ich habe doch geschrieben das ich mit hannes vorher das abgesprochen habe... und ich den teil im frontend mache .. wo ich den code so baue als würde aus dem backend **haElopage** kommen. was automatisch **false** ausgibt. 
und wenn ich feritg bin .. hannes bescheid gebe und er den teil im backend baut. 
ich habe auch **[WIP]** eingefügt ... das hätte ich gleich machen sollen. 
es gab im vorfeld eine absprache dazu .. dies hätte ich so nie gemergt.